### ISSUE TITLE
JSON parse error

### ISSUE THREAD

**Issue by Yucked** [jedijoe93/ImmortuosCalyx#6]
Version: immortuoscalyx-1.2.1.1-1.18
```java
[18:35:35] [Worker-Main-31/ERROR]: Couldn't parse data file immortuoscalyx:entities/infecteddiver from immortuoscalyx:loot_tables/entities/infecteddiver.json
com.google.gson.JsonParseException: com.google.gson.stream.MalformedJsonException: Unterminated object at line 32 column 8 path $.pools[0].entries[1].conditions
        at net.minecraft.class_3518.method_15267(class_3518.java:505) ~[server-intermediary.jar:?]
        at net.minecraft.class_3518.method_15276(class_3518.java:542) ~[server-intermediary.jar:?]
        at net.minecraft.class_4309.method_20731(class_4309.java:47) ~[server-intermediary.jar:?]
        at net.minecraft.class_4309.method_18789(class_4309.java:21) ~[server-intermediary.jar:?]
        at net.minecraft.class_4080.method_18791(class_4080.java:11) ~[server-intermediary.jar:?]
        at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) [?:?]
        at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) [?:?]
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) [?:?]
        at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) [?:?]
        at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) [?:?]
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) [?:?]
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) [?:?]
Caused by: com.google.gson.stream.MalformedJsonException: Unterminated object at line 32 column 8 path $.pools[0].entries[1].conditions
        at com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1562) ~[gson-2.8.8.jar:?]
        at com.google.gson.stream.JsonReader.doPeek(JsonReader.java:490) ~[gson-2.8.8.jar:?]
        at com.google.gson.stream.JsonReader.hasNext(JsonReader.java:413) ~[gson-2.8.8.jar:?]
        at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:725) ~[gson-2.8.8.jar:?]
        at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:718) ~[gson-2.8.8.jar:?]
        at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:726) ~[gson-2.8.8.jar:?]
        at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:718) ~[gson-2.8.8.jar:?]
        at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:726) ~[gson-2.8.8.jar:?]
        at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:701) ~[gson-2.8.8.jar:?]
        at com.google.gson.internal.bind.TypeAdapters$35$1.read(TypeAdapters.java:908) ~[gson-2.8.8.jar:?]
        at net.minecraft.class_3518.method_15267(class_3518.java:503) ~[server-intermediary.jar:?]
        ... 11 more
[18:35:35] [Worker-Main-31/ERROR]: Couldn't parse data file immortuoscalyx:entities/infectedhuman from immortuoscalyx:loot_tables/entities/infectedhuman.json
com.google.gson.JsonParseException: com.google.gson.stream.MalformedJsonException: Unterminated object at line 32 column 8 path $.pools[0].entries[1].conditions
        at net.minecraft.class_3518.method_15267(class_3518.java:505) ~[server-intermediary.jar:?]
        at net.minecraft.class_3518.method_15276(class_3518.java:542) ~[server-intermediary.jar:?]
        at net.minecraft.class_4309.method_20731(class_4309.java:47) ~[server-intermediary.jar:?]
        at net.minecraft.class_4309.method_18789(class_4309.java:21) ~[server-intermediary.jar:?]
        at net.minecraft.class_4080.method_18791(class_4080.java:11) ~[server-intermediary.jar:?]
        at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) [?:?]
        at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) [?:?]
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) [?:?]
        at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) [?:?]
        at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) [?:?]
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) [?:?]
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) [?:?]
Caused by: com.google.gson.stream.MalformedJsonException: Unterminated object at line 32 column 8 path $.pools[0].entries[1].conditions
        at com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1562) ~[gson-2.8.8.jar:?]
        at com.google.gson.stream.JsonReader.doPeek(JsonReader.java:490) ~[gson-2.8.8.jar:?]
        at com.google.gson.stream.JsonReader.hasNext(JsonReader.java:413) ~[gson-2.8.8.jar:?]
        at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:725) ~[gson-2.8.8.jar:?]
        at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:718) ~[gson-2.8.8.jar:?]
        at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:726) ~[gson-2.8.8.jar:?]
        at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:718) ~[gson-2.8.8.jar:?]
        at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:726) ~[gson-2.8.8.jar:?]
        at com.google.gson.internal.bind.TypeAdapters$29.read(TypeAdapters.java:701) ~[gson-2.8.8.jar:?]
        at com.google.gson.internal.bind.TypeAdapters$35$1.read(TypeAdapters.java:908) ~[gson-2.8.8.jar:?]
        at net.minecraft.class_3518.method_15267(class_3518.java:503) ~[server-intermediary.jar:?]
        ... 11 more
```

**Comment by CartoonishVillain** [comment]
Good catch! I'll look into it after work. Thank you for reporting.

**Comment by CartoonishVillain** [comment]
Fixed in FABRIC-1.2.1.2-1.18.1
